app/loc/lang-fi.json

```diff
@@ -8,6 +8,7 @@
     "STR_MTG_LOGIN_PROGRESS": "Haetaan kokousta",
     "STR_MTG_EXIT_PROGRESS": "Poistutaan kokouksesta",
     "STR_MTG_LOGOUT": "Kirjaudu ulos kokouksesta",
+    "STR_MTG_LOGOUT_CONFIRM": "Olet kirjautumassa ulos kokouksesta. Jatketaanko?",
```

Comment:
'Olet poistumassa ulos (...)'
or
'Haluatko varmasti poistua kokouksesta'
Can mention 'kirjautua' because users will mix it up with presence records.